client/packages/programs/src/JsonForms/common/components/Autocomplete.tsx

```diff
@@ -9,21 +9,35 @@ import { z } from 'zod';
 import { useZodOptionsValidation } from '../hooks/useZodOptionsValidation';
 import { DefaultFormRowSx, FORM_LABEL_WIDTH } from '../styleConstants';
 
-type Options = {
-  freeText?: boolean;
-  values?: string[];
-};
-const Options: z.ZodType<Options | undefined> = z
+const Options = z
   .object({
     freeText: z.boolean().optional(),
```

Comment:
freeText is currently not used and I haven't tested it. Hopefully it is still working though...

Reply:
Doesn't seem like it's working anymore...

Reply:
Doesn't seem like it's working anymore...